data/translations/en/state-data.json

```diff
@@ -1,471 +1,925 @@
 {
   "al": {
     "hp_link": "https://www.sos.alabama.gov/alabama-votes",
+    "hp_link_english_only": "false",
+    "hp_link_spanish_selection": "false",
     "registration_link": "https://www.alabamainteractive.org/sos/voter_registration/voterRegistrationWelcome.action",
+    "registration_link_english_only": "false",
+    "registration_link_spanish_selection": "false",
     "more_info_link": "https://www.sos.alabama.gov/alabama-votes/voter/register-to-vote",
+    "more_info_link_english_only": "false",
+    "more_info_link_spanish_selection": "false",
     "ip_deadline": "15 days before Election Day",
     "online_deadline": "15 days before Election Day",
     "mail_deadline": "15 days before Election Day",
-    "confirm_registration": "https://myinfo.alabamavotes.gov/VoterView/RegistrantSearch.do"
+    "mail_deadline2": "",
```

Comment:
Why do some state use `mail_deadline` and others use `mail_deadline2` can they all just use `mail_deadline`?

Reply:
**register.json**
Remove `dates__bymail_deadline2`
`dates__bymail_deadline1` change to `dates__bymail_postmarked_deadline`
`dates__bymail_deadline2_1` change to `dates__bymail_received_deadline`

**state-data.json**
`mail_deadline` change to `mail_postmarked_deadline`
`mail_dealine2` change to `mail_received_deadline`

Reply:
Update the `deadline.html` template to reflect these changes.